x-pack/solutions/observability/plugins/infra/public/pages/logs/page_content.tsx

```diff
@@ -88,7 +102,9 @@ export const LogsPageContent: React.FunctionComponent = () => {
         <RedirectWithQueryParams from={'/analysis'} to={routes.logsAnomalies.path} exact />
         <RedirectWithQueryParams from={'/log-rate'} to={routes.logsAnomalies.path} exact />
         <RedirectWithQueryParams from={'/'} to={routes.logsAnomalies.path} exact />
-        // Legacy renders and redirects
+        {i18n.translate('xpack.infra.logsPageContent.routes.LegacyRendersAndLabel', {
+          defaultMessage: '// Legacy renders and redirects',
+        })}
```

Comment:
Looks like this comment would not have been interpreted as such by the JSX compiler before and the linter was a bit over-zealous here 😁 How about this?

```suggestion
        {/* Legacy renders and redirects */}
```